SSA/Projects/InstCombine/LLVM/PrettyEDSL.lean

```diff
@@ -23,6 +23,16 @@ syntax "llvm.sub"     : InstCombine.bin_op_name
 syntax "llvm.udiv"    : InstCombine.bin_op_name
 syntax "llvm.urem"    : InstCombine.bin_op_name
 syntax "llvm.xor"     : InstCombine.bin_op_name
+syntax "llvm.icmp.eq"     : InstCombine.bin_op_name
+syntax "llvm.icmp.ne"     : InstCombine.bin_op_name
+syntax "llvm.icmp.slt"     : InstCombine.bin_op_name
+syntax "llvm.icmp.sle"     : InstCombine.bin_op_name
+syntax "llvm.icmp.sgt"     : InstCombine.bin_op_name
+syntax "llvm.icmp.sge"     : InstCombine.bin_op_name
+syntax "llvm.icmp.ult"     : InstCombine.bin_op_name
+syntax "llvm.icmp.ule"     : InstCombine.bin_op_name
+syntax "llvm.icmp.ugt"     : InstCombine.bin_op_name
+syntax "llvm.icmp.uge"     : InstCombine.bin_op_name
```

Comment:
I suspect this is broken, since the `bin_op` macro generates `($t, $t) -> $t` as the type, but `icmp` is `($t, $t) -> i1`.
Hence the comment two lines below about `icmp` probably needing its own macro